### PR TITLE
OTA-2606 Update terminology for provisioning

### PR DESCRIPTION
**Pull request by merltron**
Updating the terminology in relevant topics which referred to 'implicit' and 'automatic provisioning'.
I can't push to staging right now, but attached is a zip with the HTML output for easier review.
[_site_OTA-2606_rename_provisioning.zip](https://github.com/advancedtelematic/ats-garage-docs/files/3172355/_site_OTA-2606_rename_provisioning.zip)

@EpicNuts Are you able to quickly cast a second pair of eyes over this content?

### CODE REVIEW

**Comment by pattivacek** [comment]
When I run `grep implicit * -rIin` in this branch, I still see several references to the word "implicit" that should get replaced. Some are because of changes still pending in meta-updater, but most can be fixed now already.

**Review by EpicNuts — APPROVED** [review]
LGTM! :+1:

File: _posts/2018-07-05-client-provisioning-methods.adoc [hunk 1]
```diff
@@ -6,45 +6,40 @@
...
-// TODO MERLIN: Explain the difference between different provisioning methods, why you might want to do one or another, and what steps you need to take for each.
-
-
-
 In OTA Connect, the provisioning process ensures that the device has a certificate and a unique identifier. After a device has been provisioned, it can receive software updates. OTA Connect supports two types of provisioning:
 
-* *Automatic provisioning*
+* *Provisioning with shared credentials*
 
 Devices use temporary provisioning credentials to request permanent credentials from the server. You download the temporary provisioning credentials from the OTA Connect web app and install them on a device. Once the device registers with the server for the first time, the server assigns permanent credentials to the device. The device then uses these permanent credentials for all future transactions.
```
Comment (xaviergully) [inline _posts/2018-07-05-client-provisioning-methods.adoc:13]: Indentation is not right compared to the next paragrah

File: _posts/2018-07-05-client-provisioning-methods.adoc [hunk 2]
```diff
@@ -6,45 +6,40 @@
...
-=== Enable automatic provisioning
+=== Configure a device to provision with shared credentials
 
-To enable automatic provisioning, follow these steps:
+To configure a device to provision with shared credentials, follow these steps:
 
 . Download a link:../quickstarts/generating-provisioning-credentials.html[provisioning key].
 . Edit the following `.toml` configuration file:
 +
 link:https://github.com/advancedtelematic/aktualizr/blob/master/config/sota_autoprov.toml[config/sota_autoprov.toml]
```
Comment (xaviergully) [inline _posts/2018-07-05-client-provisioning-methods.adoc:32]: @patrickvacek, let's try to update the configure file as well and get rid of '_auto_'

File: _posts/2018-08-29-enable-implicit-provisioning.adoc [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+To provision with device credentials in production, you need to have a root CA. If you want to test this provisioning method without generating a root CA, you can simulate it with the `aktualizr-cert-provider` command.
 
-To use the `aktualizr-cert-provider` command, you must still generate a link:../quickstarts/generating-provisioning-credentials.html[provisioning key] like you would for automatic provisioning. With this method, you use the provisioning key to sign the device certificate. In production, you would use the root CA to sign the device certificate, but this method is useful for testing.
+To use the `aktualizr-cert-provider` command, you must still generate a link:../quickstarts/generating-provisioning-credentials.html[provisioning key] that your devices can share. But with this method, you use the provisioning key to sign the device certificate. In production, you would use the root CA to sign the device certificate, but this method is useful for testing.
 
-To enable simulated implicit provisioning, follow these steps: ::
+To simulate provisioning with a device certificate, follow these steps: ::
 1. Add the following lines to your local.conf:
 +
 ----
```
Comment (xaviergully) [inline _posts/2018-08-29-enable-implicit-provisioning.adoc:28]: @patrickvacek, could the configuration be updated as well, i.e getting rid of implicit ?

**Review by xaviergully — CHANGES_REQUESTED** [review]
See my comments

File: _posts/2018-07-05-client-provisioning-methods.adoc (hunk 2 above)
Comment (pattivacek) [reply _posts/2018-07-05-client-provisioning-methods.adoc:32]: Yes, of course. That has to be done in the aktualizr repo and then we can come back to this and clean it up.

File: _posts/2018-07-05-client-provisioning-methods.adoc (hunk 1 above)
Comment (merltron) [reply _posts/2018-07-05-client-provisioning-methods.adoc:13]: Fixed

**Comment by merltron** [comment]
@xaviergully I fixed the issues - do you want to do a final review? fresh HTML attached:
[_site_OTA-2606_rename_provisioning.zip](https://github.com/advancedtelematic/ats-garage-docs/files/3197406/_site_OTA-2606_rename_provisioning.zip)

**Review by pattivacek — APPROVED** [review]
Thanks, LGTM minus the URLs and meta-updater bits. Do you not want to change the names of the file(s) that have "implicit" in them?

**Review by xaviergully — APPROVED** [review]

**Comment by xaviergully** [comment]
@patrickvacek, I guess there will be another PR to update the configuration files, right ?

**Comment by pattivacek** [comment]
> I guess there will be another PR to update the configuration files, right ?

Yes, first we have to actually change them in aktualizr and then meta-updater. I'll do that as soon as I have some spare cycles.

**Comment by merltron** [comment]
> Thanks, LGTM minus the URLs and meta-updater bits. Do you not want to change the names of the file(s) that have "implicit" in them?

Thanks, no I don't want to do that yet. I don't want to risk breaking links. I'd rather do that as part of the general docs refactoring for the move to Antora.

**Pull request merged**